lib/ace/ext/papeeria/katex-previewer.coffee

```diff
@@ -423,12 +445,23 @@ define((require, exports, module) ->
           jqPopoverContainer.data().popover.tip().css(position)
       }
 
+    if not I18N?
```

Comment:
pls move this outside of `katex-previewer.coffee` (into summer-2016). We'll have messages in our messages, no need to duplicate them in the code.